ntoskrnl/ex/time.c

```diff
@@ -30,6 +30,113 @@ ULONG ExpTimerResolutionCount = 0;
 
 /* FUNCTIONS ****************************************************************/
 
+/*++
+ * If successful, this function sets the following Global Variables:
+ * ExpTimeZoneInfo
+ * ExpTimeZoneBias
+ * ExpTimeZoneId
+ *
+ * This variable is modified and returned as a result of this function
```

Comment:
I think this is now evident, via the `_Out_` annotation.

Reply:
Removed unneeded comments. Thanks.